### PR TITLE
Add a tooltip to certified badge in helm catalog

### PR DESCRIPTION
**Pull request by rohitkrai03**
**Fixes**: https://issues.redhat.com/browse/ODC-6080

**Analysis / Root cause**: No tooltip as shown for the certified badge in helm catalog.

**Solution Description**: Show a tooltip that explains the badge means its a certified helm chart. 

**Screen shots / Gifs for design review**: @openshift/team-devconsole-ux @beaumorley 

https://user-images.githubusercontent.com/6041994/123836288-286b3a80-d927-11eb-82bf-55417c001207.mov

**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [x] Edge

### CODE REVIEW

**Review by jerolimov — APPROVED** [review]
lgtm! Tested it against a cluster.

I do not add /lgtm now to wait for UX approval.

/retest

**Comment by rohitkrai03** [comment]
@jerolimov Updated PR with recommended text from PM.

**Comment by jerolimov** [comment]
Matches the latest text update in https://issues.redhat.com/browse/ODC-6080

/lgtm

**Pull request merged**
